extensions/git/package.json

```diff
@@ -932,7 +932,7 @@
           "group": "inline"
         },
         {
-          "command": "git.stageAll",
+          "command": "git.stageAllUntracked",
```

Comment:
This command would still be wrong, since users would simply like to stage all files which are *not* in the merge group. And this command would simply stage all untracked files.

Reply:
So we need to implement a new command then?

Reply:
I believe we should yes, it should be visible only when there are actually Merge changes present.

Reply:
OK. I'll try to do this. Thanks 👍

Reply:
Hello @joaomoreno 
I've updated the branch with a new command `git.stageAllButMerge` which will be visible if `git.hasMergeChanges` is true.
If I'm on the right track, I will implement the `cleanAllButMerge` command also and add any test necessary (smoke tests?)
Also, I'm not quite sure if I'm setting the context `git.hasMergeChanges` at the right place in the code or if it's the right way to implement this.
Cheers.

Reply:
Unfortunately contexts are global, so this won't work, since one can have multiple repositories open.

Let's try another approach: let's just have the `stageAll` command simply stage all but the merged changes. And then have a new command `stageAllMerge` which would stage all merge conflicting files, and we'd put that command to the right of the `MERGE CHANGES` section, what do you think?

Reply:
This sounds a better approach than mingling with global contexts. It feels like a better separation of concerns. I'll make the necessary modifications.
 But how about the naming, we keep `stageAll` name or we make it more explicit?

Reply:
We gotta keep that name, yeah, since other extensions might already depend on it. No worries about that. 👍  Looking forward to the changes!

Reply:
:+1:

Reply:
@joaomoreno @pfongkye I have opened a PR #97386 that keeps the same name `stageAll` across all the resource groups.
It's basically done based on the fact that reference for the resource group is sent to the stageAll function as a second argument. So that we can use this reference to fetch files that are needed to be staged.
let me know what do you think.